src/main/java/com/newts/newtapp/api/application/user/Login.java

```diff
@@ -25,7 +25,7 @@ public Login(UserRepository repository) {
     public Void request(RequestModel request) throws UserNotFound, IncorrectPassword {
         String username = (String) request.get(RequestField.USERNAME);
 
-        User user = userRepository.findByUsernameIgnoreCase(username).orElseThrow(UserNotFound::new);
+        User user = userRepository.findByUsername(username).orElseThrow(UserNotFound::new);
 
         if (!(user.getPassword().equals(request.get(RequestField.PASSWORD)))) {
```

Comment:
I think this line also needs to implement "BCrypt.checkpw(password, user.getPassword()" for password checking.

Reply:
Yes you are right, the reason I didnt change was because I think the login interactor will have to change with the new authentication stuff

Reply:
Got it.